jina/executors/rankers/__init__.py

```diff
@@ -39,6 +38,14 @@ class Chunk2DocRanker(BaseRanker):
     COL_DOC_CHUNK_ID = 'doc_chunk_hash'
     COL_SCORE = 'score'
 
+    required_keys = {'text'}
+    """
+    Set of required keys to be extracted from matches and query to fill the information of `query` and `chunk` meta information.
```

Comment:
The inline doctring format: https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html
```suggestion
    """set: Set of required keys to be extracted from matches and query to fill the information of `query` and `chunk` meta information.
```